packages/api/src/rpc/cennzx.ts

```diff
@@ -0,0 +1,33 @@
+import {RpcMethodOpt} from '@polkadot/jsonrpc/types';
+
+import createMethod from '@polkadot/jsonrpc/create/method';
+import createParam from '@polkadot/jsonrpc/create/param';
+
+const buyPrice: RpcMethodOpt = {
```

Comment:
Now that `buyPrice` and `sellPrice` have been defined in node side, are they contained in metadata, and can be retrieved automatically?

Reply:
yes..

Reply:
If so do we still need to decorate these methods in this file, would it be better to upgrade metadata to achieve the same goal?

Reply:
Yes, we need to decorate these methods. We are not able to grab them, as we extend from polkadot API and it supports RPC calls to module specific to substrate. Tried the other ways, and ended choosing this.

Reply:
Well, did not notice you have merged this PR. But I want to learn more and understand it deeply at this point  -- 
As you said "We are not able to grab them", but can we grab them once we upgraded metadata, as our metadata has not been updated yet?